pkg/proxy/ipvs/proxier.go

```diff
@@ -1116,6 +1121,12 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 
+	// List all addresses bound to dummy device only once
+	boundAddresses, err := proxier.ipGetter.BoundIPs()
```

Comment:
Oddly I think `bindedAddresses` sounds more correct even though `boundAddresses` is grammatically correct :P

Reply:
Yes I agree, it sounds weird. In addition we have `ListBindAddress`
What do you think? BindAddresses everywhere? BindedAddresses everywhere?

Reply:
I think `BindedAddresses` makes sense cause `BindAddress` implies "addresses I need to bind later"

Reply:
I'll do that!

Reply:
I replace "Bound" by "Binded" everywhere, definitely looks better